examples/blackjack_dqn_keras.py

```diff
@@ -0,0 +1,73 @@
+''' An example of learning a Dqqp-Q Agent on Blackjack
+'''
+
+import tensorflow as tf
+import os
+
+import rlcard
+#from rlcard.agents.dqn_agent import DQNAgent
+from rlcard.agents.dqn_agent_keras import DQNAgent
+
+from rlcard.utils.utils import set_global_seed, tournament
+from rlcard.utils.logger import Logger
+
+# Make environment
+env = rlcard.make('blackjack')
+eval_env = rlcard.make('blackjack')
+
+# Set the iterations numbers and how frequently we evaluate/save plot
+evaluate_every = 100
+evaluate_num = 10000
+episode_num = 100000
+
+# The intial memory size
+memory_init_size = 100
+
+# Train the agent every X steps
+train_every = 1
+
+# The paths for saving the logs and learning curves
+log_dir = './experiments/blackjack_dqn_keras_result/'
```

Comment:
Probably want to use `os.path` module.

Reply:
I referenced the format of the other examples, for line 30 they all wrote it this way. Do you want to change all the other examples to os.path as well?